src/ontology/obi-edit.owl

```diff
@@ -20912,6 +20912,41 @@ b) that the amount of analyte/measurand detected is over some predetermined thre
     
 
 
+    <!-- http://purl.obolibrary.org/obo/OBI_0002985 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/OBI_0002985">
```

Comment:
@beckyjackson This assay term snuck into `obi-edit.owl`, and the SPARQL "branch" tests didn't catch it. Can we update the tests to catch this case?